samples/emoji-search/values/src/commonMain/kotlin/example/values/IntervalList.kt

```diff
@@ -0,0 +1,211 @@
+/*
+ * Copyright (C) 2022 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package example.values
+
+import androidx.compose.runtime.collection.MutableVector
+import androidx.compose.runtime.collection.mutableVectorOf
+import kotlinx.serialization.Serializable
+
+/**
+ * The list consisting of multiple intervals.
```

Comment:
This is in emoji-search . . . . is that right long-term?

Reply:
I guess right now we’re pinned in that package until we have composable schemas

Reply:
It's definitely not right long term — it's already one of my TODOs in the PR description:

> Pull out the LazyColumn stuff to a separate module.